src/test/java/com/google/api/codegen/testdata/python_main_library.baseline

```diff
@@ -302,6 +315,22 @@ class LibraryServiceApi(object):
         """
         Lists shelves.
 
+        Example:
+          >>> from library.library_service_api import LibraryServiceApi
+          >>> from google.gax import CallOptions, INITIAL_PAGE
+          >>> api = LibraryServiceApi()
+          >>>
+          >>> # Iterate over all results
+          >>> for element in api.list_shelves():
+          >>>   pass
+          >>>   # process element
```

Comment:
Would prefer the comment go over the `pass` line, so that it's clear that `pass` means "process element" rather than "no-op".